database/models/user.ts

```diff
@@ -0,0 +1,49 @@
+import { models, model, Schema } from "mongoose";
+
+export enum Role {
```

Comment:
can we document somewhere (code comment or notion) what the distinction between these roles are?